checkouts-documentation.cfg

```diff
@@ -47,22 +43,23 @@ documentation               = git ${remotes-https:plone}/documentation.git egg=f
 buildout.coredev            = git ${remotes-https:plone}/buildout.coredev.git egg=false depth=1 branch=5.0
 ansible-playbook            = git ${remotes-https:plone}/ansible-playbook.git egg=false depth=1 branch=STABLE
 bobtemplates.plone          = git ${remotes-https:plone}/bobtemplates.plone.git egg=false depth=1 branch=master
-plone.api                   = git ${remotes-https:plone}/plone.api.git egg=false depth=1 branch=master
 collective.transmogrifier   = git ${remotes-https:collective}/collective.transmogrifier.git egg=false depth=1 branch=master
 
+# these are coming from plone releases, and their tags should be updated when newer Plone versions come out
+diazo                       = git ${remotes-https:plone}/diazo.git egg=false depth=1 branch=1.2.3
```

Comment:
it actually does, since git 1.8 you can checkout a tag as if it is a branch. It will put you in 'detached head' mode, meaning it's not a fully functional copy, but that's not the intention here. You just get the tagged version of the repo, with no history. Which is all we need for docs.